Source/Core/DiscIO/RiivolutionPatcher.cpp

```diff
@@ -364,148 +364,109 @@ static FSTBuilderNode* FindFileNodeInFST(std::string_view path, std::vector<FSTB
                            create_if_not_exists);
 }
 
-static void FindFilenameNodesInFST(std::vector<DiscIO::FSTBuilderNode*>* nodes_out,
-                                   std::string_view filename, std::vector<FSTBuilderNode>* fst)
+static DiscIO::FSTBuilderNode* FindFilenameNodeInFST(std::string_view filename,
+                                                     std::vector<FSTBuilderNode>& fst)
 {
-  for (FSTBuilderNode& node : *fst)
+  for (FSTBuilderNode& node : fst)
```

Comment:
Are you able to make this `const` ?

Reply:
No, because this is potentially the return value and thus must be mutable.

Reply:
Yeah, it looks like all the usages were just getting data on the item but I wasn't 100% sure.  NBD either way